src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -27,7 +27,8 @@
 from fault_config_utils import FaultInjection
 from general_utils import \
     get_partition_hosts, stop_processes, get_default_config_file, pcmd, get_file_listing, \
-    DaosTestError, run_command, get_avocado_config_value, set_avocado_config_value
+    DaosTestError, run_command, get_avocado_config_value, set_avocado_config_value, \
+    dump_engines_stacks
```

Comment:
This doesn't look like a cherry-pick.  In the master version it looks like:
```
from general_utils import \
    get_partition_hosts, stop_processes, get_default_config_file, pcmd, get_file_listing, \
    DaosTestError, run_command, dump_engines_stacks, get_avocado_config_value, \
    set_avocado_config_value
```

Reply:
hmmm, this is strange. Could I have cherry-picked a wrong commit ?